unit-42-intel-objects-feed/connector.py

```diff
@@ -0,0 +1,29 @@
+""" Copyright start
```

Comment:
Change the copyright details to 
"""
Copyright start
MIT License
Copyright (c) 2023 Fortinet Inc
Copyright end
"""